axum/src/extract/path/mod.rs

```diff
@@ -319,11 +329,19 @@ impl fmt::Display for ErrorKind {
         match self {
             ErrorKind::Message(error) => error.fmt(f),
             ErrorKind::InvalidUtf8InPathParam { key } => write!(f, "Invalid UTF-8 in `{}`", key),
-            ErrorKind::WrongNumberOfParameters { got, expected } => write!(
-                f,
-                "Wrong number of parameters. Expected {} but got {}",
-                expected, got
-            ),
+            ErrorKind::WrongNumberOfParameters { got, expected } => {
+                write!(
+                    f,
+                    "Wrong number of type parameters to `Path`. Expected {} but got {}",
```

Comment:
```suggestion
                    "Wrong number of path arguments for `Path`. Expected {} but got {}",
```

If it says parameters, expected and got are the wrong way around I think.